cypress/integration/pages/testsForAllPages.js

```diff
@@ -393,6 +397,40 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
           );
       });
     });
+
+    if (pageType === 'mediaAssetPage' || pageType === 'photoGalleryPage') {
+      describe('CPS PGL and MAP Tests', () => {
+        it('should render a timestamp', () => {
+          cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+            ({ body }) => {
+              const { lastPublished, firstPublished } = body.metadata;
+              const timeDifferenceMinutes =
+                (lastPublished - firstPublished) / 1000 / 60;
+              const minutesTolerance = 1;
```

Comment:
Did you decide against commenting to explain the need for this logic? Happy to merge if we think it's better not to comment

Reply:
Whoops, must've forgotten to copy it over when I merged the tests. I'll add it now.

Reply:
Thanks @HarveyPeachey !